### DISCUSSION TITLE
Data Tables Warning

### DISCUSSION THREAD

**Discussion by 19807311** [teamalgoritma/community#418] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/114885734/198304579-9af764c6-9d92-469f-9dd1-c14699091b67.png)

Ketika buka tab ini dan mau ketik di kolom "search", ada warning seperti ini. 
Apa yang harus dilakukan?
Terima kasih.

**Comment by jafarhf1** [comment, marked as answer]
Dear Bu bidhari @19807311,
Terkait error tersebut kemungkinan Ibu harus menambahkan parameter `locale` pada bagian fungsi `read_csv()` di dalam file `global.R` nya. Contoh penggunaannya adalah sepert berikut:
```
read_csv("data.csv", locale = locale(encoding = "latin1"))
```
Demikian yang dapat kami sampaikan, semoga membantu.

**Reply by 19807311** [reply]
![image](https://user-images.githubusercontent.com/114885734/198580129-f246a1db-8b2e-4380-9b9b-717c4c3b4998.png)

![image](https://user-images.githubusercontent.com/114885734/198580429-a65776f1-7af3-4cc9-ae04-d88996fe0a17.png)

malah tidak bisa di run, Mas Jafar.

**Reply by jafarhf1** [reply]
Halo Bu Bidhari @19807311 ,
Untuk fungsi `read_csv()` berdasarkan yang Ibu perlihatkan sepertinya tidak ada masalah. Saran saya adalah setelah menjalankan fungsi tersebut, Ibu bisa melakukan inspeksi terlebih dahulu terhadap datanya bisa menggunakan `head()` atau `glimpse()`, karena kemungkinan errornya adalah nama kolom yang ibu tulis belum sesuai dengan datanya. Semoga membantu.
